src/main/java/edu/rice/pcdp/runtime/Runtime.java

```diff
@@ -54,6 +55,14 @@ public static void resizeWorkerThreads(final int numWorkers)
         SystemProperty.numWorkers.set(numWorkers);
         taskPool = new ForkJoinPool(numWorkers);
     }
+    
+    public static void setExecutorService( ExecutorService executorService ) throws InterruptedException {
```

Comment:
A javadoc comment would be very helpful. See resizeWorkerThreads above for an example.